client-src/elements/chromedash-admin-blink-component-listing_test.ts

```diff
@@ -4,8 +4,10 @@ import {assert, expect, fixture, oneEvent} from '@open-wc/testing';
 import sinon from 'sinon';
 import {DefaultApi} from 'chromestatus-openapi';
 
-/** @type {Map<number, import('chromestatus-openapi').ComponentsUser>} */
-const testUsersMap = new Map([
+const testUsersMap = new Map<
+  number,
+  import('chromestatus-openapi').ComponentsUser
```

Comment:
Move this import up to the file level as `import type {ComponentsUser} from 'chromestatus-openapi';`